src/components/Panel.js

```diff
@@ -74,6 +75,10 @@ const Footer = () => (
         to="https://www.linkedin.com/school/founders-and-coders/"
         img={linkedin_logo_white}
       />
+      <Icon
```

Comment:
Also, check the appearance of the footer at narrow screen widths. The flexbox container for the icons gets cramped at smaller widths in the desktop view

<img width="628" alt="screen shot 2019-03-07 at 13 40 46" src="https://user-images.githubusercontent.com/28838993/53960750-fa6ee600-40de-11e9-9ff7-25e3295650df.png">

(note that the styling in this site is largely done using [tachyons](https://tachyons.io) so you'll need to change the width class on the div)

Reply:
Cheers, good spots! 

Seem to have solved the spacing at desktop/`ns` width, would also like your thoughts on mobile:

![](http://i.imgur.com/fg10WuB.png)

Can't tell if the instagram and linkedin icons are unevenly spaced, or if it's just to do with the fact that they're both full squares (and so look closer together)? 

Even if the latter is the case, do you think it'd still be worth giving a little extra margin between them?

Thanks

Reply:
Actually sorry just seen that the spacing is too much at desktop width :face_with_head_bandage: will change again